site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -118,17 +148,42 @@ Register icons in the manifest under the `"icons"` field.
 
 ## Additional UI features {: #additional_features }
 
+### Action badge {: #badge }
+
+Badges display a colored banner on top of the action icon. They can only be used when the `"action"`
+is declared in the manifest. 
+
+Use badges to indicate the state of the extension. The [Drink Water][sample-drink] extension sample
+displays a badge with "ON" to show the user they successfully set an alarm and displays nothing when
+the extension is idle. Badges can contain up to 4 characters.
+
+{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/nXwAHSWLBEgT8099ITT0.png", alt="Badge On",
+       height="72", width="72" %}
+
+{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/pNz8UgfTBMmcf7fE9wja.png", alt="Badge Off",
+       height="72", width="72" %}
+
+You can set the text of the badge by calling [`chrome.action.setBadgeText`][action-setbadgetext] and
+the banner color by calling
+[`chrome.action.setBadgeBackgroundColor`][action-setbadgebackgroundcolor].
+
+```js
+chrome.action.setBadgeText({text: 'ON'});
+chrome.action.setBadgeBackgroundColor({color: '#4688F1'});
+```
+
 ### Popup {: #popup }
 
-A popup is an HTML file that is displayed in a special window when the user clicks the toolbar icon.
+A popup is an HTML file that is displayed in a special window when the user clicks the action icon.
 A popup works very similarly to a web page; it can contain links to stylesheets and script tags, but
-does not allow inline JavaScript.
+does not allow inline JavaScript. The popup cannot be smaller than 25x25 px nor larger than 800x600
+px.
```

Comment:
Size requirements are also covered on the Action API page. Rather than include it here can we cross-link so there's only one canonical page with this info?

https://developer.chrome.com/docs/extensions/reference/action/#popup